app/src/main/java/com/sunlightlabs/congress/models/Roll.java

```diff
@@ -33,10 +33,10 @@
     // if a bill is associated
     public String bill_id, bill_title;
 
-
-    // not yet migrated
 	public Map<String,Vote> voters;
-	public Map<String,Vote> voter_ids;
```

Comment:
All that `voter_ids` was used for was to look up a specific member's position, never a whole array. This was collapsed to the field `member_position` below, used only when needed.